src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  --large-icon-size: 19px;
+  --large-icon-margin-left: 26px;
+  --small-icon-size: 11px;
+  --small-icon-margin-left: 13px;
+}
+
+.search-form {
+  position: relative;
+}
+
+.search-form:first-child {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-icon {
+  position: absolute;
+}
+
+.search-icon-large {
+  top: 25px;
+  left: var(--large-icon-margin-left);
+}
+
+.search-icon-small {
+  height: var(--small-icon-size);
+  width: var(--small-icon-size);
+  top: 15px;
+  left: var(--small-icon-margin-left);
+}
+
+.search-bar {
+  width: 100%;
+  font-family: Avenir, serif ;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+}
+
+.search-bar-large {
+  height: 70px;
+  font-size: 16px;
+  padding-left: calc(var(--large-icon-margin-left)
+  + var(--large-icon-size) + 17px);
+}
+
+.search-bar-small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: calc(var(--small-icon-margin-left)
+  + var(--small-icon-size) + 9px);
+}
```

Comment:
There should be search-bar, which contains default search bar styles and modified `search-bar--small` or `search-bar--big`, which differs only by it's sizes from default search-bar